hie-ghc-tree/Haskell/Ide/GhcTreePlugin.hs

```diff
@@ -24,7 +24,7 @@ ghcTreeDescriptor = PluginDescriptor
   , pdCommands =
 
         buildCommand trees (Proxy :: Proxy "trees") "Get ASTs for the given file"
-                   [".hs",".lhs"] (SCtxFile :& RNil) RNil
+                   [".hs",".lhs"] (SCtxFile :& RNil) RNil ChangeCurrent
```

Comment:
Similar to above. If you have added some code, it may not return everything you are looking for.